packages/server/populate-local-db.sh

```diff
@@ -0,0 +1,11 @@
+#! /bin/bash
+set -a; source .env; set +a
+psql -h $LOCAL_IP -p $DB_PORT -d $DATABASE_NAME -U $DATABASE_USER << END_OF_SCRIPT
+TRUNCATE "CategoryDataContents", "CategoryDataBase" restart identity;
+
+INSERT INTO "CategoryDataBase" ("rataextraRequestPage", "alfrescoFolder", "writeRights")
+VALUES ('/linjakaaviot', '/123', 'Rataextra_kirjoitus_linjakaaviot');
+
+INSERT INTO "CategoryDataContents" ("baseId", "fields")
+VALUES (1, '{"test": "test"}')
+END_OF_SCRIPT
```

Comment:
We could seed our database using TS [also](https://www.prisma.io/docs/guides/database/seed-database#example-seed-scripts).

But this approach works too 👍🏻

Reply:
Definitely something to improve upon. And then there's the matter of where to store the dev-env values for population. Not sure if we want to have those in public like this, if they'd be populated from here.